tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -318,6 +320,31 @@ final class BuildRunner extends Runner {
     return result.ok;
   }
 
+  Future<void> _postGn() async {
+    if (dryRun) {
+      return;
+    }
+    final Engine engine = Engine.fromSrcPath(engineSrcDir.path);
+    final Output? latest = engine.latestOutput();
+    if (latest == null) {
+      return;
+    }
+    final io.File commandsFile = latest.compileCommandsJson;
```

Comment:
I'd suggest directly computing the path to `compile_commands.json` here using something like `path.join(engineSrcDir.path, 'out', build.ninja.config, 'compile_commands.json')` instead of relying on timestamps. It probably can't happen in this simple case, but I've seen enough tests start randomly flaking due to referencing file timestamps on the real file system that this code gives me the heebie-jeebies.

Reply:
I was doing exactly that yesterday, but build.ninja.config might not be there. Was concerned that was hacky; but I can put a check in and put it back.